validate.py

```diff
@@ -87,7 +91,7 @@ def validate_updates(user, updates, auth_usr = None):
     #quick utilities: to reject any update or any update by a non-admin.
     say_no = lambda x, y, z: False
     def say_no_to_non_admin(x, y, z):
-        return auth_usr['role']['organizer'] or auth_usr['role']['director']
+        return auth_usr['role']['organizer'] or auth_usr['role']['director'] #this will return either true or false. it is only true if organizer or director can be found
```

Comment:
See above... This is better phrased as `This returns true iff the auth_user is a director or organizer`, but this is literally what the code says.